oasislmf/preparation/correlations.py

```diff
@@ -25,6 +26,19 @@ def map_data(data: Optional[dict]) -> Optional[pd.DataFrame]:
         supported_perils_df = pd.DataFrame(supported_perils)
         correlation_settings_df = pd.DataFrame(correlation_settings)
 
+        if len(correlation_settings_df) > 0:
+            print(correlation_settings_df)
```

Comment:
remove if left over print otherwise I think we should use logger.info or debug

Reply:
good catch, it was a debug print leftover.